.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+pretty-quick --staged
```

Comment:
I have installed https://github.com/azz/pretty-quick to run prettify on staged files
as found on https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick